prow/github/client.go

```diff
@@ -226,11 +229,14 @@ func NewClient(getToken func() []byte, bases ...string) *Client {
 //   An endpoint is used when all preceding endpoints have returned a conn err.
 //   This should be used when using the ghproxy GitHub proxy cache to allow
 //   this client to bypass the cache if it is temporarily unavailable.
-func NewDryRunClient(getToken func() []byte, bases ...string) *Client {
+// 'graphqlEndpoint' is the Github GraphQL API (v4) endpoint to use.
+//   The only difference between githubql.NewEnterpriseClient and
```

Comment:
Not sure we need to keep this comment here

Reply:
I'll remove them, no problem.